pom.xml

```diff
@@ -96,7 +96,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-lambda-java-log4j</artifactId>
-            <version>1.0.0</version>
+            <version>1.0.1</version>
```

Comment:
So this doesn't contain any fix this is from 2020, according to this: https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-log4j2 we should be using `1.5.1`

Reply:
My bad. Fixed.